src/mui/input/ReferenceManyInput.js

```diff
@@ -0,0 +1,222 @@
+import React, {Component, PropTypes} from 'react';
```

Comment:
CS: please add spaces inside brackets. `{Component, PropTypes}` => `{ Component, PropTypes }`

This happens in many places in this PR.